source/github-helpers/index.ts

```diff
@@ -34,9 +34,35 @@ export const getCurrentCommittish = (pathname = location.pathname, title = docum
 		return parsedTitle.groups!.branch;
 	}
 
+	// TODO [2021-04-12]: .last needed for #2799 // GHE
```

Comment:
```suggestion
	// TODO [2021-08-12]: .last needed for #2799 // GHE
```

Reply:
Why?

Reply:
https://github.com/sindresorhus/refined-github/runs/2376926124

Reply:
Oh god. Hah I meant to write the same day, but in 2022

Reply:
Can you also add `unicorn/expiring-todo-comments: warn` in the xo settings? I don't want it to automatically fail lint.